qdrant/v1.1.x/distributed_deployment.md

```diff
@@ -501,3 +501,30 @@ client.upsert(
     ordering="strong"
 )
 ```
+
+
+## Listener mode
+
+<aside role="alert">This is an experimental feature, its behavior may change in the future.</aside>
+
+In some cases it might be useful to have a qdrant node that only accumulates data and does not participate in search operations.
+There are several scenarios where this can be useful:
+
+- Listener can be used to store data in a separate cluster, which can be used for backup purposes or to store data for a long time.
```

Comment:
I am a bit confused with the `separate cluster` bit. 
Is a listener node part of two clusters?

Reply:
I thinks it's typo. Should be node